R/Library.r

```diff
@@ -241,6 +245,10 @@ SPACox.one.SNP = function(g,
   pos.na = which(is.na(g))
   missing.rate = length(pos.na)/N
 
+  if(missing.rate == 1){
+    return(c(NA, 1, NA, NA, NA, NA, NA))
+  }
+

```

Comment:
This portion is the actual fix for the missingness bug--the rest is all cleanup of documentation and imports.